index.php

```diff
@@ -13,10 +13,10 @@
 	// Check if there are inputs
 	if (isset($_POST['dbs-database'], $_POST['dbs-search']))
 	{
-	    $search = $_POST['dbs-search'];
+    $search = $_POST['dbs-search'];
 		$_SESSION['query'] = $_POST['dbs-search'];
 		$_SESSION['database'] = $database = $_POST['dbs-database'];
-		$strict = ($_POST['dbs-strict']) ? TRUE : FALSE;
+		$strict = (isset($_POST['dbs-strict'])) ? TRUE : FALSE;
```

Comment:
Better to use !empty here.

Reply:
i think that `isset` is better that `!empty` when checking if an array key exists. But I can change it to `!empty` if you insist. Or, even better, use [array_key_exists](https://www.php.net/manual/en/function.array-key-exists.php)

Reply:
It can exist but still being false. As we want to be strict, it's better to use empty function here.